frontend/src/core/services/PocketUserService.js

```diff
@@ -135,6 +135,57 @@ class PocketUserService extends PocketBaseService {
       });
   }
 
+    /**
+   * Login user with email.
+   *
+   * @param {string} username Username of user to login.
+   * @param {string} password Password of user.
+   *
+   * @return {string} Message about error or empty if there's none
+   */
+  validateLogin(username, password) {
```

Comment:
The validation of data of each case(in this case the login), should be in the view component.